tests/utils_tests/test_text.py

```diff
@@ -95,17 +95,72 @@ def test_truncate_chars(self):
         )
 
     def test_truncate_chars_html(self):
-        perf_test_values = [
```

Comment:
Some of these are not valid HTML so don't get through the HTML parser. e.g 50,000 `&`s. I'd need a bit of help to understand what exactly this is testing?

Reply:
If it's not valid, I wouldn't worry. These were likely just stress testing the old parser to avoid ReDOS issues.

Reply:
These tests have since been re-written in 17b51094d778b421bb2b3aae0c270894b050455d